source/OptimaJet.DWKit.StarterApplication/Services/HttpOrganizationContext.cs

```diff
@@ -13,11 +13,11 @@ public class HttpOrganizationContext : IOrganizationContext
 
         private bool parsed;
         private bool hasOrganization;
+        private bool isOrganizationHeaderPresent;
```

Comment:
if the header is omitted, we do not filter at all. (well, we fallback to repo.get())